tests.py

```diff
@@ -37,6 +37,8 @@ def tearDownClass(self):
     "F2": 0,
     "h0": 5,
     "cosi": 0,
+    "psi": 0,
+    "phi0": 0,
```

Comment:
It's actually `phi` :wink:

Reply:
Argh. That's a completely needless case of lal->PyFstat divergence... Future wishlist item: let's stop randomly renaming things. ;)

Fixing this fixes all tests except for the new one and one other (see below). For the new one, I guess we need to call the usual translation dict somewhere.

Could you meanwhile take a look at:
```
tests.py::TestBinaryModulatedWriter::test_tp_parsing - ValueError: 'psi' is not in list
```

Reply:
I'm on it

Reply:
Looks like the test suite does not do what I think it did, since e.g. binary arguments are not included in `BinaryModulatedWriter` and so... I'll check that after fixing this.